CHANGELOG.md

```diff
@@ -19,6 +19,150 @@ const expandWidget = new Expand({
 })
 ```
 
+### View Popup deprecations and changes
+In a continuous effort to optimize the performance of the API, more specifically the load time, the decision was made to stop bundling the Popup with the MapView and SceneView. The Popup widget receives new features regularly, such as when browsing related records. Each new feature added to the popup widget increases the amount of built code and size of the widget that is being sent to the web browser. For example, the Popup module, which is bundled with the MapView, represents around 50% of the size of the bundle.
+
+In this release, the Popup widget loading is deferred until the view is ready and will only be loaded if there are layers with a popup configured since it is only useful once content is displayed on the view. Please expand the details below for breaking changes and the migration strategy.
+
+TODO: Add code examples.
+
+<details>
+  <summary>Click to expand the details regarding the migration strategy and breaking changes for the view's popup</summary>  
+
+### Scenarios that still work
+
+If using any of the following scenarios, your applcation will not break:
+  
+#### Setting popup options:
+```js
+const view = new MapView({
+  popup: {
+    dockEnabled: true
+  }
+});
+view.popup.dockEnabled = true;
+```
+
+#### Opening & closing the popup:
+- It is recommended to use the new `View.openPopup()` method instead of `view.popup.open()`, however, it will still work.
+```js
+// prompts a deprecation warning if popup isn't created
+// calls view.openPopup() under the hood
+view.popup.open(...);
+
+// new
+view.openPopup(...);
+```
+- It is recommended to use the new `View.closePopup()` method instead of `view.popup.close()`, however, it will still work.
+```js
+// prompts a deprecation warning if popup isn't created
+// calls view.closePopup() under the hood
+view.popup.close(...);
+
+// new
+view.closePopup();
+```
+
+#### Disable the view's popup
```

Comment:
looks like you were using colons in the other headers
```suggestion
#### Disable the view's popup:
```

Reply:
Headers shouldn't need colons.

Reply:
was just going for consistency! but good point - @lboyd93 could you remove the colons from each of the headers?